src/main/kotlin/com/meltwater/docker/compose/DockerCompose.kt

```diff
@@ -201,7 +201,8 @@ class DockerCompose private constructor(
     private fun verifyDockerComposeInstallation() {
         try {
             val versionOutput = executeCommand("docker-compose --version")
-            val v: MatchResult? = Regex(".* (\\d+\\.\\d+\\.\\d+)").find(versionOutput)
```

Comment:
This could be `".* v?(\\d+\\.\\d+\\.\\d+)"` to optionally match the v. Output for me on Linux is `Docker Compose version 2.2.3` so that would correctly parse both Linux and Mac output.